### PR TITLE
Merge pbs fix

### PR DESCRIPTION
**Pull request by lars-petter-hauge**

### CODE REVIEW

File: ert_shared/ensemble_evaluator/config.py [hunk 1]
```diff
@@ -7,7 +7,6 @@
 CLIENT_URI = "client"
 DISPATCH_URI = "dispatch"
 DEFAULT_PORT = str(8765 + random.randint(0, 30))
-DEFAULT_HOST = "0.0.0.0"
```
Comment (sondreso) [inline ert_shared/ensemble_evaluator/config.py:10]: ?

Comment (lars-petter-hauge) [reply ert_shared/ensemble_evaluator/config.py:10]: As you might spot, this was not a successful operation :D

The merge is not done correctly, probably doing the inverse of what I want.

Comment (lars-petter-hauge) [reply ert_shared/ensemble_evaluator/config.py]: Fixed

File: ert_shared/ensemble_evaluator/config.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 logger = logging.getLogger(__name__)
 CLIENT_URI = "client"
 DISPATCH_URI = "dispatch"
-DEFAULT_PORT = str(8765 + random.randint(0, 30))
+DEFAULT_PORT = str(8765 + random.randint(0, 100))
```
Comment (jondequinor) [inline ert_shared/ensemble_evaluator/config.py:9]: This should be done in some other way. Unix already offers `0` as a way to get a random port.

Comment (sondreso) [reply ert_shared/ensemble_evaluator/config.py:9]: Yes, and also this is not the problem it turns out :sweat_smile:

**Comment by sondreso** [comment]
This PR can be closed, I will rebase ert3 instead!

**Pull request closed without merging**